### ISSUE TITLE
Artillery test on Azure ACI failed to conclude

### ISSUE THREAD

**Issue by mmathoosh** [artilleryio/artillery#3355]
Version info: 
```
<"^2.0.20",>
```

Running this command: 
```
"artilleryRunInAzure": "artillery run-aci --variables \"{\\\"passwordPerfTest\\\": \\\"xxxx\\\"}\" xxxx.yml --record --key xxxxxx --tenant-id xxxxxx --subscription-id xxxxxxx --storage-account artillexxxaccount --blob-container artilleryxxxblobcontainer --resource-group xxx --region northeurope --count 1"
```

I expected to see this happen:

at the end of the tests I'd expect that test is finalized

Instead, this happened:

test stays in the stage on the picture
![image](https://github.com/user-attachments/assets/1032e893-e9d3-4418-ad3a-62d6051756a2)
![image](https://github.com/user-attachments/assets/e102da25-a6cb-4031-93f7-3479bf9fd558)

**Comment by mmathoosh** [comment]
Additional information: "Page performance" metrics are not uploaded during/after the test run to the artillery.app.io
![image](https://github.com/user-attachments/assets/ff163736-351b-43fd-ba30-e8f7dc3c420f)

**Comment by hassy** [comment]
thank you @mmathoosh, we'll look into it. In the second screenshot - is that from a different test?

**Comment by mmathoosh** [comment]
> thank you @mmathoosh, we'll look into it. In the second screenshot - is that from a different test?

yes, it is different test. I have suspicion something from these steps failed: 
4. Collect logs and metrics from running container groups and aggregate and print reports (**also missing in the container**)
5. Delete AQS queue, files uploaded to ABS, and ACI container groups when the test is finished

**Comment by hassy** [comment]
@mmathoosh we'll need more details to try to debug what might have gone wrong. Would you mind dropping me an email on hveldstra@artillery.io or sending a message via the chat widget in the Artillery Cloud dashboard? Would like to fix this asap
